pkg/webhooks/pod_webhook.go

```diff
@@ -139,6 +139,11 @@ func (p *PodWebhook) Default(ctx context.Context, obj runtime.Object) error {
 			return err
 		}
 	}
+
+	if err := podutils.AddLWSVariables(pod); err != nil {
```

Comment:
itt it may simplify some use cases (init scripts) to have this be available to the leader containers as well. I don't know if that is the most compelling use case, but imo it seems more convenient to have this standardized across all containers in the lws.

Reply:
Why not localhost then?

Reply:
localhost is not relevant here :)